.github/workflows/stale-planning-issue.yml

```diff
@@ -0,0 +1,54 @@
+name: 'Close stale Planning issues and PRs'
```

Comment:
Planning is intended including PM tasks. It's more like a backlog, probably we should rename it.

We don't need to do any cleanup, as it's intended. However, reviewing issues in the backlog should be a regular process. This would be tackled by a different approach.

Reply:
okay, I'll remove this.